crates/prune/src/pruner.rs

```diff
@@ -130,25 +158,25 @@ impl<DB: Database> Pruner<DB> {
     /// Prunes the segments that the [Pruner] was initialized with, and the segments that needs to
     /// be pruned according to the highest static_files.
```

Comment:
```suggestion
    /// Prunes the segments that the [Pruner] was initialized with, and the segments that needs to
    /// be pruned according to the highest static_files. Segments are parts of the database that represent one or 
    /// more tables.
```
copied that text from one of your previous review comments, it's very helpful